src/components/StatusIndicator/StatusStep/__tests__/StatusStep.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file Status step tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { StatusStep } from '../../../..';
+
+import { STATUS } from '../StatusStep';
+
+describe('StatusStep', () => {
+  // eslint-disable-next-line compat/compat
+  Object.values(STATUS).forEach(status =>
```

Comment:
This is because of Opera Mini support 🤔

Reply:
As a fallback, I believe we can do the following - 

```suggestion
  Object.keys(STATUS).map(status => STATUS[status]).forEach(status =>
```

Reply:
Sure, but the opera mini issue comes up elsewhere as well:
<img width="732" alt="Screen Shot 2020-03-23 at 10 20 37 AM" src="https://user-images.githubusercontent.com/9057921/77332341-f7e22d00-6cef-11ea-8316-1895f8ebb86f.png">

I suggest we update the compat rules instead. 🤔